recipes/pdspy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pdspy" %}
+{% set version = "1.5.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 94d68282132f0272ef41fa4da84eebbf859ead2f3cb407223feeb9f2d1a7a247
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+    skip: True  # [win]
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - {{ compiler('fortran') }}
+    host:
+        - python
+        - setuptools
```

Comment:
You have pip, so you can remove setuptools in this list

Reply:
Fixed